stunnel/defaults.yml

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
Well, there's a reason these were called `service_defaults`. They are the default settings for each single service config in `conf.d/`. They are _not_ part of the "global" stunnel config.

I tested your formula on Ubuntu 16.04. These settings didn't even get applied.